fix for http://sourceforge.net/tracker/index.php?func=detail&aid=1056397&group_id=6212&atid=106212
(but lvm support still sucks - partimage displays /dev/dm-N instead of 
/dev/volume_group/logical_volume)
--- partimage-0.6.7/src/client/misc.cpp.orig	2009-01-15 23:02:06.646830497 +0100
+++ partimage-0.6.7/src/client/misc.cpp	2009-01-15 23:03:21.641676275 +0100
@@ -1889,7 +1889,7 @@
     cPtr++;
   memset(szDevice, 0, 127);
   cCur = szDevice;
-  while(isalnum(*cPtr) || *cPtr == '/')
+  while(isalnum(*cPtr) || *cPtr == '/' || *cPtr == '-')
     {
       *cCur = *cPtr;
       cCur++;
